src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_ITypeOfExpression.vb

```diff
@@ -1,4 +1,4 @@
-' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
```

Comment:
Same here, BOM is expected but was previously removed (https://github.com/dotnet/roslyn/issues/22319)